react/components/ProductSummaryPrice/ProductSummaryPrice.js

```diff
@@ -34,6 +44,38 @@ const ProductSummaryPrice = ({
     sellingPriceClass: 'dib ph2 t-body t-heading-5-ns',
   }
 
+  const sortPrices = (priceA, priceB) => priceA - priceB
+  const lowestPrice = compose(head, sort(sortPrices))
+  const highestPrice = compose(last, sort(sortPrices))
+
+  const getListPrices = prices => {
+    const lowPrice = lowestPrice(prices)
+    const highPrice = highestPrice(prices)
+    return [
+      lowPrice,
+      highPrice,
+    ]
+  }
+
+  const isAvailableProduct = price => price !== 0
+
+  const getPriceRange = () => {
+    const items = prop('items', product)
+    if (items) {
+      const sellers = flatten(map('sellers', items))
+      const prices = map(path(['commertialOffer', 'Price']), sellers)
+      const availableProductsPrices = filter(isAvailableProduct, prices)
+      
+      return getListPrices(availableProductsPrices)
+    }
+
+    return []
+  }
+
+  const priceRange = getPriceRange()
+  const [lowPrice, highPrice] = priceRange
+  const showPriceRange = lowPrice !== highPrice
```

Comment:
Just to be super safe, please put:
`const showPriceRange = priceRange.length === 2 && lowPrice !== highPrice`